src/autopas/utils/SimilarityFunctions.h

```diff
@@ -0,0 +1,106 @@
+/**
+ * @file SimilarityFunctions.h
+ * @author J. Kroll
+ * @date 01.06.2020
+ */
+
+#pragma once
+
+#include "ThreeDimensionalMapping.h"
+#include "autopas/containers/ParticleContainerInterface.h"
+#include "autopas/molecularDynamics/MoleculeLJ.h"
```

Comment:
```suggestion
```

Not needed anymore as far as I can see.

Reply:
resolved by commit 5d64489f8fe659f14ef7b5bbc52c69f6219fb882